tooling/bundler/src/bundle/windows/templates/update-task.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-16"?>
+<!-- 
+  Copyright 2019-2021 Tauri Programme within The Commons Conservancy
+  SPDX-License-Identifier: Apache-2.0
+  SPDX-License-Identifier: MIT
+-->
+<!-- Copyright 2019-2021 Tauri Programme within The Commons Conservancy
```

Comment:
oops ;)

Reply:
It's double-protected ;-)

Jk, I just push the fix.